src/posix/platform/system.cpp

```diff
@@ -82,20 +82,20 @@ static void processStateChange(otChangedFlags aFlags, void *aContext)
 }
 #endif
 
-static const char *get802154RadioUrl(otPlatformConfig *aPlatformConfig)
+static const char *get802154RadioUrl(const char *aRadioUrls[], uint8_t aRadioUrlNum)
```

Comment:
We may set the `otPlatformRadioUrls ` as a member of the `otPlatformConfig`. The func `get802154RadioUrl(onst char *aRadioUrls[], uint8_t aRadioUrlNum) ` could be changed to `get802154RadioUrl(const otPlatformRadioUrls &aRadioUrls)`.

Reply:
My concern is that the change of `otPlatformConfig` will break the existing app code. For example, `ot-br-posix` needs to be updated accordingly. But I think let's do the change and update `ot-br-posix` accordingly.